subworkflows/local/igv_tracks.nf

```diff
@@ -0,0 +1,42 @@
+//
+// IGV TRACKS
+//
+
+include { JUNCTION_TRACK   } from '../../modules/local/junction_track'
+include { UCSC_WIGTOBIGWIG } from '../../modules/nf-core/ucsc/wigtobigwig/main'                                                                          
+
+workflow IGV_TRACKS {
+    take:
+        wig
+        path_sizes
+        gene_counts
```

Comment:
The channel contains both, and gene_counts was its original name in the workflow. I didn't change it to avoid confusion, but you are right. I could change it to something like junctions_and_counts